src/groupeddataframe/grouping.jl

```diff
@@ -400,8 +386,12 @@ When computing the `sum` or `mean` over floating point columns, results will be
 accurate than the standard [`sum`](@ref) function (which uses pairwise summation). Use
 `col => x -> sum(x)` to avoid the optimized method and use the slower, more accurate one.
 
-### Examples
+[`by(f, df, cols)`](@ref) is a shorthand for `combine(f, groupby(df, cols))`.
```

Comment:
Why not keep this in "See also"?

Reply:
I thought "See also" was not used in Base. Also this shortens the printout of a docstring. I can revert "See also" sections if you prefer.

Reply:
Yes, "See also" is mentioned at https://docs.julialang.org/en/latest/manual/documentation/ (I added it :-).

That sounds better to me since this line gives information which isn't actually relevant for `combine`: it's relevant for `by`.

Reply:
Ah - now I see what you meant. Fixed